man/ifcfg-dhcp.5.in

```diff
@@ -71,6 +71,10 @@ one of them does it.
 .BR DHCLIENT_ROUTE_PRIORITY
 This option allows to set a metric/priority for DHCPv4 routes. Default is 0.
 .TP
+.BR DHCLIENT_ROUTE_SET_SRC\ { \fIno\fR* | yes }
+When set to yes, the DHCP IP address is set as the preferred source
+preferred source in dhcp routes.
```

Comment:
```suggestion
When set to yes, the DHCP IP address is set as the preferred source 
in dhcp routes.
```

Reply:
fixed, but also changed to use DHCPv4 instead dhcp/DHCP.